chapters/102/index.rst

```diff
@@ -0,0 +1,202 @@
+================================================
+Jina 102: How basic components work together
+================================================
+
+
+In Jina 102, you’ll learn how Jina’s basic components work together. Before you start, make sure you’ve already read through `Jina 101`_.
+
+.. _Jina 101: https://101.jina.ai 
+
+
+.. contents:: Table of Contents
+    :depth: 3
+
+
+Search workflows
+==================
+
+
+First, we’ll show you how a :term:`Flow` works in Jina. Be aware, there are many different ways to build a Flow, and this is just one way.
+
+Let’s use a simple text-to-text search application as an example — searching for sentences in a book. We use two Flows for this:
+
+*   An **indexing** Flow makes the whole book searchable by sentence
+*   A **querying** Flow handles a user query (in this case, a sentence) and returns search results
+
+A working example can be found `here`_. (Bear in mind that this example uses pre-processed sentences, so the segmenting step is skipped)
+
+The indexing Flow prepares data to be searched. Input Documents are fed in, processed, and output at the other end as searchable indexes in storage:
+
+.. _here: https://github.com/jina-ai/examples/tree/master/wikipedia-sentences
+
+|
+
+.. image:: /chapters/102/images/index_flow.png 
+	:width: 80%  
+
+|
+
+
+Then, the querying Flow takes a user query as its input Document, and returns a list of ranked matches:
+
+|
+
+.. image:: /chapters/102/images/query_flow.png 
+	:width: 80%  
+
+|
+
+Inside a Flow
+-----------------
+
+Here, we’ll use the indexing Flow as an example:
+
+|
+
+.. image:: /chapters/102/images/flow_pods.png
+	:width: 80%   
+
+|
+
+In our Flow, different :term:`Pods<Pod>` perform different tasks. Documents (in this case a book) are:
+
+1. Input Documents segmented into chunks (sentences)
+2. Chunks transformed into vector embeddings
+3. Chunks indexed in key-value pairs and saved to storage
+
+Inside a Pod
+----------------------
+
+|
+
+.. image:: /chapters/102/images/pod_peas.png
+	:width: 80%  
+
+|
+
+In the Segmenter Pod, there’s only one :term:`Pea`. But sometimes you group multiple copies of the same Peas in a Pod to increase reliability or performance. To implement sharing and parallelization, check `this section`_.
+
+.. _this section: link to sharing section
+
+
+Inside a Pea
+----------------------
+
+
+Let’s take a closer look at the Pea in the Segmenter Pod:
+
+
+|
+
+.. image:: /chapters/102/images/pea_exec_driver.png 
+	:width: 60%  
+
+|
+
+As you can see, a Pea is a wrapper for an :term:`Executor` and its Driver. 
+
+While Peas and Pods perform the communication in a Flow, the messages themselves come from the Executors. And each Executor needs a specific Driver to handle its input and output messages.
```

Comment:
the messages are processed by Executors may be more accurate than saying they come from?